indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -566,6 +567,23 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_avg_per_question_per_track(self):
```

Comment:
Do we really need to return the scores for all tracks? If it doesn't incur in massive performance drawback I would rather have `get_questions_score(self, track)`. I would compose the per-track dictionary outside.